test/e2e/framework/ingress_utils.go

```diff
@@ -997,6 +1104,26 @@ func (j *IngressTestJig) ConstructFirewallForIngress(gceController *GCEIngressCo
 	return &fw
 }
 
+func (j *IngressTestJig) DetectDistinctResponse() (int, error) {
+	// Wait for the loadbalancer IP.
+	address, err := WaitForIngressAddress(j.Client, j.Ingress.Namespace, j.Ingress.Name, LoadBalancerPollTimeout)
+	if err != nil {
+		Failf("Ingress failed to acquire an IP address within %v", LoadBalancerPollTimeout)
+	}
+	response := sets.NewString()
+	timeoutClient := &http.Client{Timeout: IngressReqTimeout}
+
+	for i := 0; i < 100; i++ {
+		res, err := SimpleGET(timeoutClient, fmt.Sprintf("http://%v", address), "")
+		if err != nil {
+			Logf(res)
```

Comment:
Put a message or something here, this will look like a lot of log spam.

Reply:
ack